web-app/src/app/router/Router.tsx

```diff
@@ -23,6 +24,7 @@ export const AppRouter: React.FC = () => {
         <Route path='account' element={<Account />} />
       </Route>
       <Route path='contact-info' element={<ContactInformation />} />
+      <Route path='forgot-password' element={<ForgotPassword />} />
```

Comment:
should use a protected route here
<Route element={<ProtectedRoute />}> ... </Route>